features/docker.feature

```diff
@@ -0,0 +1,285 @@
+Feature: Command output from box command
+  service-manager should return the correct output from box commands
+
+  Scenario Outline: Boot and execute box commands
+    Given box is <box>
+    And provider is <provider>
+    And a file named "Vagrantfile" with:
+    """
+    begin
+      require 'vagrant-libvirt'
+    rescue LoadError
+      # NOOP
+    end
+
+    Vagrant.configure('2') do |config|
+      config.vm.box = '<box>'
+      config.vm.box_url = 'file://../../.boxes/<box>-<provider>.box'
+      config.vm.network :private_network, ip: '<ip>'
+      config.vm.synced_folder '.', '/vagrant', disabled: true
+      config.servicemanager.services = 'docker'
+    end
+    """
+
+    When I run `bundle exec vagrant up --provider <provider>`
+    Then the exit status should be 0
```

Comment:
We can include this too here to verify message 
https://github.com/projectatomic/vagrant-service-manager/blob/master/features/adb-openshift.feature#L29-L34